### PR TITLE
NO MERGE: Proof-of-concept showing neutrino, TC preset, and Yarn

### PR DESCRIPTION
**Pull request by eliperelman**
DO NOT MERGE!

This PR is meant as a proof of concept. It demonstrates using neutrino and neutrino-preset-taskcluster out of the box. It also demos using Yarn to eliminate npm shrinkwrap issues.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -516,16 +515,5 @@ let load = base.loader({
 
 }, ['profile', 'process']);
 
-// If this file is executed launch component from first argument
-if (!module.parent) {
-  load(process.argv[2], {
```
Comment (jonasfj) [inline src/index.js]: I'm pretty sure this part would still have been needed :)

File: test/queueservice_test.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ suite('queue/QueueService', function() {
   var Promise       = require('promise');
   var slugid        = require('slugid');
   var assert        = require('assert');
-  var QueueService  = require('../lib/queueservice');
+  var QueueService  = require('../src/queueservice');
```
Comment (jonasfj) [inline test/queueservice_test.js]: Does neutrino do path rewriting?

File: src/index.js (hunk 1 above)
Comment (eliperelman) [inline src/index.js]: This was moved to root/index to simplify the call and separate the bin from the module.

File: test/queueservice_test.js (hunk 2 above)
Comment (eliperelman) [inline test/queueservice_test.js]: It can, but it doesn't. Using mocha and babel with Node builds the source files.

**Pull request closed without merging**